[HCCDOC#1765]: Add recommendations-filtering content to Remote Health Monitoring (#70364)
openshift/openshift-docs: modules/removing-filters-from-insights-advisor-recommendations.adoc

// Module included in the following assemblies:
//
// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc

:_mod-docs-content-type: PROCEDURE
[id="removing-filters-from-insights-recommendations_{context}"]
= Removing filters from Insights Advisor recommendations

> You can apply multiple filters to the list of recommendations and remove them individually or completely reset them.

Reviewer: ```suggestion
You can apply multiple filters to the list of recommendations or remove them individually. You can also reset the filters.
```
- Edit for minimalism and to eliminate the awkward grammatical construction.

Author: done

.Removing filters individually

> * Where the applied filters are displayed, remove applied filters individually, including the default filters, by clicking the *X* icon next to the filter.

Reviewer: ```suggestion
* Where the applied filters are displayed, remove applied filters individually, including the default filters, by clicking the *X* icon next to the filter.
```
- Rewrite as a step in a procedure.
- Reword to use [the ISG guidance on referring to UI elements](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui). Not sure if this is a drop-down list but the description needs to use the standard UI element wording for whatever it is.

Author: It's not a drop-down list. Each selected filter looks like an ovaltangular chip with a tiny x in the upper-left corner of it. All of the selected filters are visible at once. To remove individual filters, you click the x on the filter "chip." To remove them all (except for the defaults), you click (the blue words that aren't a button) *Reset filters*.

I rewrote each of these as procedures, but still used bullets since they're single steps.

.Removing all non-default filters

> * You can remove only the filters that you applied, leaving the default filters in place, by clicking *Reset filters*.

Reviewer: ```suggestion
* Click *Reset filters* to remove all the filters that you applied and to leave the default filters in place.
```
- Rewrite as a step in a procedure.

Author: done
